src/index.ts

```diff
@@ -0,0 +1,311 @@
+/// <reference path="index.d.ts" />
+
+document.addEventListener('DOMContentLoaded', () => {
+    const page = new Page();
+
+    page.getDataEvents().then(() => {
+        new PointerEventsDom();
+    });
+
+    adaptiveMenu.showMenu();
+});
+
+const templates = {
+    'card-item'(domNode: HTMLElement, data: TemplateData) {
+        let icon = <HTMLImageElement>domNode.querySelector('.card-item__icon');
```

Comment:
До перевода на ts все переменные объявлялись с `var`. 
При переводе на ts ты в некоторых местах везде используешь let, а в некоторых везде const.

В этом месте линтер ругается:
```
[tslint] Identifier 'icon' is never reassigned; use 'const' instead of 'let'.
```

Поставь линтер, если его еще нет и сможешь быстро все исправить (большую часть таких мест можно поправить https://palantir.github.io/tslint/usage/cli/, смотреть на --fix)

Reply:
На этой же строке:
```
[tslint] Type assertion using the '<>' syntax is forbidden. Use the 'as' syntax instead.
```